cli/internal/codegen/client.go

```diff
@@ -52,7 +52,7 @@ func Client(lang Lang, appSlug string, md *meta.Data) (code []byte, err error) {
 	}()
 
 	var gen generator
-	switch lang {
```

Comment:
Since `lang` is effectively an enumeration it’s unexpected to change its value like this. Can we move this logic to the place where the value is parsed instead?

Reply:
The issue is where we parse it, we simply convert to the `Lang` type from what ever raw input was given. This is the first time we do any validation on the data - hence why I kept it with the validation logic.

Happy to move, but then I think we should fully validate the enum there, rather than just convert the data type

Reply:
Yeah, I think we should validate it where we do the conversion so we don't end up with `Lang` values that are invalid in the first place